api/Request.json

```diff
@@ -97,75 +53,31 @@
           "description": "<code>Request()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "41",
+              "version_added": "42",
```

Comment:
Good, removing flags reveals this kind of error in the data. I presume the lint was failing and that's how you noticed?

Reply:
You guessed it!  This seems to be pretty common among both flagged and prefixed data.

Reply:
With a prefixed parent feature it should still usually be correct, since interface and member are rarely (never?) both prefixed, but yeah there are similar issues.